### PR TITLE
Make pack install work with local git repo

### PR DESCRIPTION
**Pull request by dzimine**
- Make st2 pack.install work with file:///path/to/local/git/repo.
- Add unit tests to cover permutations.

### CODE REVIEW

**Comment by dzimine** [comment]
I was missing this too much on my "pack dev workflow testing". See if it's safe enough to shove into 2.1, which I think it is.

File: contrib/packs/actions/pack_mgmt/download.py [hunk 1]
```diff
@@ -278,12 +278,15 @@ def _eval_repo_url(repo_url):
         """Allow passing short GitHub style URLs"""
         if not repo_url:
             raise Exception('No valid repo_url provided or could be inferred.')
-        has_git_extension = repo_url.endswith('.git')
-        if len(repo_url.split('/')) == 2 and "git@" not in repo_url:
-            url = "https://github.com/{}".format(repo_url)
+        if "file:" in repo_url:
```
Comment (emedvedev) [inline contrib/packs/actions/pack_mgmt/download.py]: Let's do `file://` and `startswith` just to be certain.

Comment (dzimine) [inline contrib/packs/actions/pack_mgmt/download.py]: Right. Done. Thanks!

**Comment by Kami** [comment]
LGTM :+1:

**Pull request merged**